pkg/convertx/ptr.go

```diff
@@ -0,0 +1,5 @@
+package convertx
```

Comment:
Is there an other convert package that the name collides with? That's why we generally use the `x` suffix.